### PR TITLE
Cu 33dj0z4 add song owners UI jonatan escobar

### PR DESCRIPTION
**Pull request by escobarjonatan**
**What:**
- Create UI for adding song owners/creditors

**Notes:**
- Since there is some unknown on the data the backend is expecting from us i did not want to get too deep into the data and focused on the UI.

**Screenshot:**
image

image

[Mocks](https://www.figma.com/file/zrn01ZR2zFeLiKBebEYI1v/Artist-Portal---Final-Designs?node-id=605%3A5644)

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-33dj0z4 Add song owners UI](https://app.clickup.com/t/33dj0z4)

File: src/assets/images/CloseCircleFill.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
 const CloseCircleFill = () => (
-  <svg width="20" height="20" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <path d="M9.00002 17.3327C4.39752 17.3327 0.666687 13.6018 0.666687 8.99935C0.666687 4.39685 4.39752 0.666016 9.00002 0.666016C13.6025 0.666016 17.3334 4.39685 17.3334 8.99935C17.3334 13.6018 13.6025 17.3327 9.00002 17.3327ZM9.00002 7.82102L6.64335 5.46352L5.46419 6.64268L7.82169 8.99935L5.46419 11.356L6.64335 12.5352L9.00002 10.1777L11.3567 12.5352L12.5359 11.356L10.1784 8.99935L12.5359 6.64268L11.3567 5.46352L9.00002 7.82102Z" fill="#FF453A"/>
+  <svg
+    width="20"
+    height="20"
```
Comment (escobarjonatan) [inline src/assets/images/CloseCircleFill.tsx:4]: I dont know why it shows this way in the PR but the file looks good 🤷🏽‍♂️

File: src/pages/home/Home.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const Home: FunctionComponent = () => {
   const [isMobileOpen, setMobileOpen] = useState(false);
   return (
     <Box
-      sx={ { display: "flex", flexGrow: 1, background: theme.colors.black100 } }
+      sx={ { backgroundColor: theme.colors.black100, display: "flex", flexGrow: 1, height: "fit-content" } }
```
Comment (escobarjonatan) [inline src/pages/home/Home.tsx:30]: This line fixes the issue of the Home container not expanding the full height when content is more than the fold

File: src/pages/home/uploadSong/MintSong.tsx [hunk 3]
```diff
@@ -59,51 +60,55 @@ const MintSong: FunctionComponent = () => {
...
-          sx={ {
-            marginTop: [theme.spacing(2), theme.spacing(2), 0],
-          } }
-          value={ isModalOpen ? "on" : "off" }
-          checked={ values.isMinting }
-          onChange={ (event) => {
-            setIsModalOpen(event.target.value === "off");
-          } }
-        />
+        <AddOwner />
```
Comment (escobarjonatan) [inline src/pages/home/uploadSong/MintSong.tsx:99]: I decided to break down the owner logic into its own component so we dont blow this file up

**Comment by escobarjonatan** [comment]
> Looks good! One change is that the "Add owner" UI should only appear if the user has toggled minting the song on. Also, it's not in the mock-ups, but I think the modal should have a "Cancel" button to the left of the "Add" button. It may not be intuitive enough that users can click outside of the modal. I've left a comment on the Figma mock-up suggesting the change.

@scandycuz  Good catch on the owner display only if minting, got carried away with the testing. 
I have added a button bc i do agree it's not as clear for the user, here's what it looks like now:

image

**Pull request merged**